test/unit/node_presenter_test.rb

```diff
@@ -74,7 +74,7 @@ def teardown
       state.phrases = PhraseList.new(:four, :one, :two, :three)
       presenter = NodePresenter.new("flow.test", outcome, state)
 
-      Airbrake.expects(:notify_or_ignore).once
+      Rails.logger.expects(:warn).with("[Missing phrase] The phrase being rendered is not present: flow.test.phrases.four. Responses: ").once
```

Comment:
This is a bit of a bad example, but in real life this will log the path that leads to the outcome that has missing phrases.